index.js

```diff
@@ -342,6 +342,25 @@ module.exports = {
     },
 
     /**
+     * Call this if you plan on loading TypeScript files.
+     *
+     * Encore.configureTypeScript(function(tsConfig) {
+     *      // change the tsConfig
+     * });
+     *
+     * Supported configuration options:
+     * @see https://github.com/TypeStrong/ts-loader/blob/master/README.md#available-options
+     *
+     * @param {function} callback
+     * @return {exports}
+     */
+    configureTypeScript(callback) {
+        webpackConfig.configureTypeScript(callback);
```

Comment:
Oh, and one more thing: let's rename this to `enableTypeScriptLoader()` to be consistent with the other loaders (`configureBabel()` is the outlier - because Babel is always enabled, so we're *just* configuring it).